### PR TITLE
Implement Node Clustering

### PR DESCRIPTION
**Pull request by OhxSteve**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
+const cluster = require('cluster');
 const routes = require('./../routes');
+
 require('newrelic');
 
 const app = express();
 
-app.set('port', process.env.PORT || 3001);
+if (cluster.isMaster) {
+  const numWorkers = require('os').cpus().length;
```
Comment (allisonc07) [inline server/app.js]: require statements should be up at the top! so after line 4 would be good!

Comment (OhxSteve) [reply server/app.js]: Good catch! Thanks @allisonc07 I've gone ahead and removed the white space

**Review by allisonc07 — CHANGES_REQUESTED** [review]
Looks good! Left one comment for you to change!

**Comment by OhxSteve** [comment]
@allisonc07 Thanks! Refactor done

**Review by allisonc07 — APPROVED** [review]

**Pull request merged**